### PR TITLE
fix bdf parsing fpgabist

### PR DESCRIPTION
**Pull request by buie1**
Improved BDF parsing to fix issue when trying to find fpga on different platforms.

### CODE REVIEW

**Comment by buie1** [comment]
Are there any things you want to address on this PR, @OPAE/tools

File: tools/extra/pac/fpgabist/fpgabist [hunk 1]
```diff
@@ -44,7 +44,7 @@ def main(args):
         sys.exit("No FPGA devices found")
     elif len(bdf) > 1 and not any([args.bus, args.device, args.function]):
         sys.exit("Multiple cards found. Specify a bus, device or function")
-    else:
+    elif vars(args)['bus'] or vars(args)['device'] or vars(args)['function']:
```
Comment (dasm) [inline tools/extra/pac/fpgabist/fpgabist]: This one can be written as:
```
elif any(vars(args)['bus'], vars(args)['device'], vars(args)['function'])
```

File: tools/extra/pac/fpgabist/bist_common.py [hunk 2]
```diff
@@ -35,6 +35,8 @@
 REQ_CMDS = ['lspci', 'fpgainfo', 'fpgaconf', 'fpgadiag', 'fpga_dma_test',
             'bist_app']
 
+ROOT_PATH = '/sys/class/fpga'
+FPGA_DEVICE = os.path.join(ROOT_PATH, 'intel-fpga-dev.{instance_id}')
```
Comment (dasm) [inline tools/extra/pac/fpgabist/bist_common.py]: This doesn't seem to be anyhow connected to this change.
Am I missing something?

Comment (buie1) [reply tools/extra/pac/fpgabist/bist_common.py]: Good catch. this was unnecessary. I had started to do a cleanup will address in later PR

**Comment by buie1** [comment]
@OPAE/tools @asgibbons @dasm 
Want to double back to get this merged in.
Any thing else we want to get resolved?

**Pull request merged**